### PR TITLE
Adding Dark mode

### PR DESCRIPTION
**Pull request by TCVinNYC**
Give users the option for Light, Dark, or Adaptive Theming

### CODE REVIEW

**Comment by TCVinNYC** [comment]
Currently only Adaptive Theming works, please feel free to add additional suggestions for how things should look

File: mobile/ios/Runner.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -361,7 +361,7 @@
 				CODE_SIGN_IDENTITY = "Apple Development";
 				CODE_SIGN_STYLE = Automatic;
 				CURRENT_PROJECT_VERSION = 35;
-				DEVELOPMENT_TEAM = 2F67MQ8R79;
```
Comment (alextran1502) [inline mobile/ios/Runner.xcodeproj/project.pbxproj:364]: I don't think we want to include this file, do we?

**Comment by alextran1502** [comment]
Can you help adding some dark theme screenshot to this PR?

**Comment by alextran1502** [comment]
I try to run it on an Android device when toggling the switch it doesn't change the color.

**Comment by Coyote6705** [comment]
> Can you help add some dark theme screenshots to this PR?

Hey Alex! I don't know if you still need them, but here are some dark theme screenshots (on an android device)
Screenshot_20220805-141331
Screenshot_20220805-140123
Screenshot_20220805-140247
Screenshot_20220805-140746
Screenshot_20220805-140736
Screenshot_20220805-140225
Screenshot_20220805-140233
Screenshot_20220802-230314

**Comment by alextran1502** [comment]
Impelemted in #468

**Pull request closed without merging**